packages/now-node-bridge/src/bridge.ts

```diff
@@ -0,0 +1,164 @@
+import { AddressInfo } from 'net';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { Server, IncomingHttpHeaders, OutgoingHttpHeaders, request } from 'http';
+
+interface NowProxyEvent {
+  Action: string;
+  body: string;
+}
+
+interface NowProxyRequest {
+  isApiGateway: boolean;
+  method: string;
+  path: string;
+  headers: IncomingHttpHeaders;
+  body: Buffer;
+}
+
+interface NowProxyResponse {
+  statusCode: number;
+  headers: OutgoingHttpHeaders;
+  body: string;
+  encoding: string;
+}
+
+function normalizeEvent(event: NowProxyEvent | APIGatewayProxyEvent): NowProxyRequest {
+  let method: string;
+  let path: string;
+  let body: string | null;
+  let encoding: string;
+  let headers;
+  let isApiGateway = true;
+  let bodyBuffer: Buffer | null = null
+
+  if ((event as NowProxyEvent).Action === 'Invoke') {
```

Comment:
```suggestion
  if ('Action' in event && event.Action === 'Invoke') {
```

Try to avoid casting, let the runtime checks guide the type system like a Jedi uses the Force 😉

Reply:
@TooTallNate Actually, not I'm not convinced this is correct.

What happens if the event has an `Action` but its action is not `Invoke`?